src/main/java/com/openlattice/datastore/search/controllers/SearchController.java

```diff
@@ -608,7 +616,7 @@ public SearchResult executeFQNPropertyTypeSearch( @RequestBody FQNSearchTerm sea
                 } )
         );
 
-        List<AuditableEvent> events = new ArrayList<>( neighborsByEntitySet.keySet().size() + 1 );
+        List<AuditableEvent> events = Lists.newArrayListWithExpectedSize( neighborsByEntitySet.keySet().size() + 1 );
```

Comment:
Same as above with `WithExpectedSize` vs `WithCapacity`/`ArrayList<>` constructor. You know the exact size in this case